frontend/app/views/testing/testUsers.scala.html

```diff
@@ -37,6 +37,24 @@
                         <li>
                             <a href="javascript:
                             (function() {
+                                if (!document.getElementById('register_field_username')) {
```

Comment:
This should go in a separate script like the existing bookmarklet - this is so you can update it without users needing to manually delete & recreate their bookmarklets